packages/@react-spectrum/list/test/ListViewDnd.test.js

```diff
@@ -67,9 +67,9 @@ describe('ListView', function () {
 
   beforeAll(function () {
     user = userEvent.setup({delay: null, pointerMap});
-    offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
-    offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
-    scrollHeight = jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 40);
+    // offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
+    // offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
+    // scrollHeight = jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 40);
```

Comment:
The DnD tests can be run if we mock the following lines:
https://github.com/adobe/react-spectrum/blob/7c7e84624629d4963ff500cbd75a3412cb3a9e8f/packages/%40react-aria/virtualizer/src/useVirtualizerItem.ts#L48 to a defined value
and
https://github.com/adobe/react-spectrum/blob/7c7e84624629d4963ff500cbd75a3412cb3a9e8f/packages/%40react-stately/virtualizer/src/Virtualizer.ts#L145 changing the `offset.x` value to something that isn't NaN so that https://github.com/adobe/react-spectrum/blob/7c7e84624629d4963ff500cbd75a3412cb3a9e8f/packages/%40react-stately/layout/src/ListLayout.ts#L558 doesn't resolve to NaN. 

The `offset.x` needs to be non-zero for reasons that aren't clear to me yet, changing https://github.com/adobe/react-spectrum/blob/7c7e84624629d4963ff500cbd75a3412cb3a9e8f/packages/%40react-stately/virtualizer/src/Virtualizer.ts#L526 to provide a non Nan/non-zero `contentOffsetX` value to `_setContentOffset` has the same effect as changing line 145

Reply:
it is arguable whether or not we would want to mock these lines, mocking the virtualizer item's scrollWidth and scrollHeigh could make some sense so long as the values used aren't necessarily asserted against